addons/binding/org.openhab.binding.robonect/src/test/java/org/openhab/binding/robonect/handler/RobonectHandlerTest.java

```diff
@@ -0,0 +1,302 @@
+package org.openhab.binding.robonect.handler;
```

Comment:
kudos for writing tests! 👍 
please move these into a separate test fragment though